src/components/Loading.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
```

Comment:
Make this:
```
import React from 'react'
import Loader from 'halogen/PulseLoader';

export default () => (
    <Loader color="#233e5e" size="16px" margin="4px"/>
)
```

Reply:
I decided to find a loader I actually like and implement rather than relying on you guys. Please add this to your PR - should take all of 30 seconds.

Reply:
Anything for you my Benevolent Dictator ;)